src/providers/clojure.rs

```diff
@@ -54,6 +55,23 @@ impl Provider for ClojureProvider {
 }
 
 impl ClojureProvider {
+    fn has_lein_ring_plugin(&self, app: &App) -> bool {
+        self.is_using_lein(app)
+            && app
+                .read_file("project.clj")
+                .unwrap()
```

Comment:
Don't explicitly unwrap incase the `project.clj` file does not exist
```suggestion
                .unwrap_or_default()
```

Reply:
Thanks, I'm a total Rust newbie. 😄